lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -508,6 +508,10 @@ private function gatherColumn(
         // the 'default' option can be overwritten here
         $options = $this->gatherColumnOptions($mapping) + $options;
 
+        if (isset($mapping['enumType'])) {
```

Comment:
I feel this code should be moved into `gatherColumnOptions`, because that is where it should come from.